source/ProSymbolEditor/Models/SymbolAttributes/SymbolAttributeSet.cs

```diff
@@ -309,7 +442,139 @@ public void PopulateRowBufferWithAttributes(ref RowBuffer rowBuffer)
                 rowBuffer["countrycode"] = LabelAttributes.CountryCode;
             }
         }
-        
+
+        public void PopulateFeatureWithAttributes(ref Feature feature)
+        {
+            if (!string.IsNullOrEmpty(DisplayAttributes.Identity))
+            {
+                feature["identity"] = DisplayAttributes.Identity;
```

Comment:
Given the number of places these are used - I'm wondering if these attribute names should be variables/constants defined once - in case we need to change the name -or- they will differ 2525D to C to B